README.md

```diff
@@ -289,6 +291,47 @@ const meta = extractMetadata(props);
     },
     isRequired: true
   },
+  'Optional Slider': {
+    objMeta: {
+        Min: {
+            propName: 'uriBase',
```

Comment:
all the `propName` don't match. Seems to be copy pasta

Reply:
whoops! good catch 😳